nebula_decoders/src/nebula_decoders_continental/decoders/continental_ars548_decoder.cpp

```diff
@@ -0,0 +1,574 @@
+
+    assert(detection.range.value() >= 0.f && detection.range.value() <= 1500.f);
+    assert(detection.range_std.value() >= 0.f && detection.range_std.value() <= 1.f);
+    assert(detection.range_rate_std.value() >= 0.f && detection.range_rate_std.value() <= 1.f);
+
+    detection_msg.invalid_distance = detection.invalid_flags & 0x01;
+    detection_msg.invalid_distance_std = detection.invalid_flags & 0x02;
+    detection_msg.invalid_azimuth = detection.invalid_flags & 0x04;
+    detection_msg.invalid_azimuth_std = detection.invalid_flags & 0x08;
+    detection_msg.invalid_elevation = detection.invalid_flags & 0x10;
+    detection_msg.invalid_elevation_std = detection.invalid_flags & 0x20;
+    detection_msg.invalid_range_rate = detection.invalid_flags & 0x40;
+    detection_msg.invalid_range_rate_std = detection.invalid_flags & 0x80;
+    detection_msg.rcs = detection.rcs;
+    detection_msg.measurement_id = detection.measurement_id.value();
+    detection_msg.positive_predictive_value = detection.positive_predictive_value;
+    detection_msg.classification = detection.classification;
+    detection_msg.multi_target_probability = detection.multi_target_probability;
+    detection_msg.object_id = detection.object_id.value();
+    detection_msg.ambiguity_flag = detection.ambiguity_flag;
+
+    detection_msg.azimuth_angle = detection.azimuth_angle.value();
+    detection_msg.azimuth_angle_std = detection.azimuth_angle_std.value();
+    detection_msg.elevation_angle = detection.elevation_angle.value();
+    detection_msg.elevation_angle_std = detection.elevation_angle_std.value();
+
+    detection_msg.range = detection.range.value();
+    detection_msg.range_std = detection.range_std.value();
+    detection_msg.range_rate = detection.range_rate.value();
+    detection_msg.range_rate_std = detection.range_rate_std.value();
+  }
+
+  detection_list_callback_(std::move(msg_ptr));
+
+  return true;
+}
+
+bool ContinentalARS548Decoder::ParseObjectsListPacket(
+  const std::vector<uint8_t> & data, const std_msgs::msg::Header & header)
+{
+  auto msg_ptr = std::make_unique<continental_msgs::msg::ContinentalArs548ObjectList>();
+  auto & msg = *msg_ptr;
+
+  ObjectListPacket object_list;
+  assert(sizeof(ObjectListPacket) == data.size());
+
+  std::memcpy(&object_list, data.data(), sizeof(object_list));
+
+  msg.header.frame_id = sensor_configuration_->base_frame;
+
+  if (sensor_configuration_->use_sensor_time) {
+    msg.header.stamp.nanosec = object_list.stamp.timestamp_nanoseconds.value();
+    msg.header.stamp.sec = object_list.stamp.timestamp_seconds.value();
+  } else {
+    msg.header.stamp = header.stamp;
+  }
+
+  msg.stamp_sync_status = object_list.stamp.timestamp_sync_status;
+  assert(msg.stamp_sync_status >= 1 && msg.stamp_sync_status <= 3);
+
+  const uint8_t number_of_objects = object_list.number_of_objects;
+
+  msg.objects.resize(number_of_objects);
+
+  if (radar_status_.timestamp_sync_status == "SYNC_OK") {
```

Comment:
This would be clearer with a comment like "Estimate rate of dropped objects"

Reply:
Added a comment in [da81bd6](https://github.com/tier4/nebula/pull/123/commits/da81bd619f2f346d53cd6daedc885fc4929ab4f3)